content/kotlin/concepts/Objects/Objects.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'objects'
+Description: 'Concept for objects declaration in Kotlin language'
+Subjects: 
+  - 'Mobile Development'
+Tags:
+  - 'android'
+  - 'kotlin'
+  - 'object'
```

Comment:
```suggestion
  - 'Objects'
```

Reply:
Thanks. I gonna work on all of that right now.